supabase/schema/init.sql

```diff
@@ -195,7 +195,8 @@ ALTER TABLE "public"."user_shelter_badges" OWNER TO "postgres";
 CREATE TABLE IF NOT EXISTS "public"."users" (
```

Comment:
Rio's PR is splitting init.sql to sql for each table. You might have to fix this later.

Reply:
Okay. Got it